colibre/auto_plotter/snia_rates.yml

```
@@ -0,0 +1,325 @@
+    start: 1e6
+    end: 1e12
+  y:
+    quantity: "snia_rates.snia_rates_50_kpc"
+    units: "1/year"
+    start: 1e-8
+    end: 1e-1
+  median:
+    plot: true
+    log: true
+    adaptive: true
+    number_of_bins: 30
+    start:
+      value: 1e6
+      units: Solar_Mass
+    end:
+      value: 1e12
+      units: Solar_Mass
+  metadata:
+    title: Stellar Mass-SNIa rate relation (50 kpc aperture)
+    caption: Uses a 50 kpc 3D aperture.
+    section: SNIa Rate
+
+stellar_mass_snia_rates_per_stellar_mass_50:
+  type: "scatter"
+  legend_loc: "upper left"
+  x:
+    quantity: "apertures.mass_star_50_kpc"
+    units: Solar_Mass
+    start: 1e6
+    end: 1e12
+  y:
+    quantity: "derived_quantities.snia_rate_per_stellar_mass_50_kpc"
+    units: "1/year/Solar_Mass"
+    start: 1e-15
+    end: 1e-11
+  median:
+    plot: true
+    log: true
+    adaptive: true
+    number_of_bins: 30
+    start:
+      value: 1e6
+      units: Solar_Mass
+    end:
+      value: 1e12
+      units: Solar_Mass
+  metadata:
+    title: Stellar Mass-SNIa rate per stellar mass relation (50 kpc aperture)
+    caption: Uses a 50 kpc 3D aperture.
+    section: SNIa Rate
+
+stellar_mass_snia_rates_per_stellar_mass_active_only_50:
+  type: "scatter"
+  legend_loc: "upper left"
+  selection_mask: "derived_quantities.is_active_50_kpc"
+  x:
+    quantity: "apertures.mass_star_50_kpc"
+    units: Solar_Mass
+    start: 1e6
+    end: 1e12
+  y:
+    quantity: "derived_quantities.snia_rate_per_stellar_mass_50_kpc"
+    units: "1/year/Solar_Mass"
+    start: 1e-15
+    end: 1e-11
+  median:
+    plot: true
+    log: true
+    adaptive: true
+    number_of_bins: 30
+    start:
+      value: 1e6
+      units: Solar_Mass
+    end:
+      value: 1e12
+      units: Solar_Mass
+  metadata:
+    title: Stellar Mass-SNIa rate per stellar mass relation (active only, 50 kpc aperture)
+    caption: Uses a 50 kpc 3D aperture.
+    section: SNIa Rate
+
+stellar_mass_snia_rates_per_stellar_mass_passive_only_50:
+  type: "scatter"
+  legend_loc: "upper left"
+  selection_mask: "derived_quantities.is_passive_50_kpc"
+  x:
+    quantity: "apertures.mass_star_50_kpc"
+    units: Solar_Mass
+    start: 1e6
+    end: 1e12
+  y:
+    quantity: "derived_quantities.snia_rate_per_stellar_mass_50_kpc"
+    units: "1/year/Solar_Mass"
+    start: 1e-15
+    end: 1e-11
+  median:
+    plot: true
+    log: true
+    adaptive: true
+    number_of_bins: 30
+    start:
+      value: 1e6
+      units: Solar_Mass
+    end:
+      value: 1e12
+      units: Solar_Mass
+  metadata:
+    title: Stellar Mass-SNIa rate per stellar mass relation (passive only, 50 kpc aperture)
+    caption: Uses a 50 kpc 3D aperture.
+    section: SNIa Rate
+
+gas_metallicity_snia_rates_per_stellar_mass_active_only_50:
+  type: "scatter"
+  legend_loc: "upper left"
+  selection_mask: "derived_quantities.is_active_50_kpc"
+  x:
+    quantity: "derived_quantities.gas_o_abundance_fromz_avglin_50_kpc"
+    units: dimensionless 
```

Please insert the following

```suggestion
    units: dimensionless 
    log: false
```
otherwise there will be a log scale applied to log (O/H)